docs/Installation.md

```diff
@@ -1,7 +1,6 @@
 # Installation
 
-The supported and best way to install Homebrew is on the
-[homepage](https://brew.sh).
+Instructions for installing Homebrew are on the [homepage](https://brew.sh).
```

Comment:
Can we keep the word "supported" here somewhere? Thanks!

Reply:
Would the current wording work? If not, how about "Instructions for a supported install of Homebrew are on the homepage?

Reply:
The latter sounds great ❤️

Reply:
@MikeMcQuaid updated